test/functional/feature_assumeutxo.py

```diff
@@ -100,14 +100,19 @@ def expected_error(log_msg="", rpc_details=""):
         # The height is not used for anything critical currently, so we just
         # confirm the manipulation in the error message
         bogus_height = 1337
-        signed_overflow_height = 3275262676
         for bad_block_hash in [bogus_block_hash, prev_block_hash]:
-            for bad_height in [bogus_height, signed_overflow_height]:
```

Comment:
In  51f197bd84916c494e9250926776b9efc3225100 "Assumeutxo: Sanitize block height in assumeutxo metadata "
This change was added in 98e119da35c4f7e74f0c423974d497e350c8e9fa and then removed in this commit. Why not just add it like this in the previous commit with the same error `msg` string, and then update the `msg` string in this commit?

Reply:
See the description: "The first commit shows how to reproduce the error shown in https://github.com/bitcoin/bitcoin/issues/30514 and how it could be fixed with a minimal change. The second commit adds an explicit sanitizer that will probably be the preferred approach. I can squash the commits but first I would like to hear conceptual feedback since https://github.com/bitcoin/bitcoin/issues/30514 suggested to remove the block height instead."

Reply:
So I was still hunting for the conceptual consensus and didn't change this yet. I think we are coming to an end though (see my comment below). It would be great if you could indicate where you stand on this though @ismaelsadeeq . If more support for the approach here I will squash the commits and implement the suggested improvements.

Reply:
@fjahr, I reviewed the code primarily to address PRs targeting issues in the v28.0 milestone. I haven’t gone through all the conceptual discussions here, as I’m still getting up to speed with the direction of the assumeutxo project.